build.gradle

```diff
@@ -213,6 +213,7 @@ dependencyManagement {
 dependencies {
   def withoutJunit4 = {
     exclude group: 'junit', module: 'junit'
+    exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
```

Comment:
vintage is only include with spring boot test package but i guess it's fine to have all the way - no breaking denomination